packages/fx-core/templates/core/v3Migration/upgradeReport.md

```diff
@@ -11,7 +11,7 @@ Visit https://aka.ms/teams-toolkit-5.0-upgrade to learn more about the upgrade.
 1. Created `teamsapp.yml` and `teamsapp.local.yml` in your project root folder.
 2. Moved environment files in `.fx` to `.env.{env}` in `env` folder.
 3. Moved `templates/appPackage` to `appPackage` and updated placeholders in it per latest tooling's requirement, and rename `manifest.template.json` to `manifest.json`.
-4. Moved `templates/appPackage/aad.template.json` to `aad.manifest.json` and updated placeholders in it per latest tooling's requirement.
+4. If your project contains `templates/appPackage/aad.template.json`, moved it to `aad.manifest.json` and updated placeholders in it per latest tooling's requirement.
```

Comment:
suggest wording update:
"If your project contains `templates/appPackage/aad.template.json`, moved it to `aad.manifest.json` and updated placeholders in it per latest tooling's requirement." -> "If your project contains file `templates/appPackage/aad.template.json`, it will be moved and renamed as `aad.manifest.json` and placeholders in it will be updated per the latest Teams Toolkit requirement."

Reply:
Thanks, updated. I also updated another line based on your suggestion. Please help review again.